src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,24 @@
 const generalCSQuiz = [
+  {
+    Question: "If you forget one semicolon or misplace a character, the entire program will not run and send you back what error?",
+    Answer: "Syntax Error",
+    Distractor1: "Runtime Error",
+    Distractor2: "Logical Error",
+    Distractor3: "Semantic Error",
+    Explanation: "A syntax error occurs when the programmer writes an instruction using incorrect syntax. ",
```

Comment:
Please avoid using direct quotes from 3rd party resources like this one
https://www.sciencedirect.com/topics/engineering/syntax-error

```suggestion
    Explanation: "A syntax error is when the programmer writes invalid code that violates the rules of that programming language.",
```